api/all/src/main/java/io/opentelemetry/api/metrics/DefaultMeter.java

```diff
@@ -371,7 +359,6 @@ public LongGaugeBuilder setDescription(String description) {
 
     @Override
     public LongGaugeBuilder setUnit(String unit) {
-      ValidationUtil.checkValidInstrumentUnit(unit);
```

Comment:
These validation utils can be moved to the SDK now. `checkInstrumentName` probably makes most sense in `SdkMeter`. `checkValidInstrumentUnit` should be removed given that the spec has clarified that SDKs aren't supposed to do any validation on unit. You can do that removal in this PR if you want, or just move the method to `AbstractInstrumentBuilder`.

Reply:
Bumping this @breedx-splk.

Reply:
Ok, I moved `checkValidInstrumentName()` to a new internal utility class in the sdk, rather than `SdkMeter`. This seemed a little more SRPy to me.

I did move the unit check over to the `AbstractInstrumentBuilder` class, and I think removal of that should be tackled in a separate PR.

Once those were moved out of the validator, the only things that were left in`ValidationUtil` were the logging methods, so I renamed it to `ApiUsageLogger`.